packages/sdks/src/blocks/util.ts

```diff
@@ -1,6 +1,5 @@
-export function markSerializable<FN extends (...args: any[]) => any>(
+export function changeFunctionToString<FN extends (...args: any[]) => any>(
   fn: FN
-): FN {
-  (fn as any).__qwik_serializable__ = true;
-  return fn;
+): string {
+  return `return (${fn.toString()}).apply(this, arguments)`;
```

Comment:
note to self: this is the same as `serializeFn` in `src/functions/register-component.ts`, can dedupe.

Reply:
Actually, this function is simply making sure we serialize the fn ahead of time, `serializeFn` is no longer necessary